tools/pipelines/templates/build-npm-client-package.yml

```diff
@@ -550,6 +550,7 @@ extends:
           - job: Coverage_tests
             displayName: "Coverage tests"
             dependsOn: build
+            timeoutInMinutes: 25 # Normally takes around 20 minutes. Decreasing timeout to speed up iteration while I find which tests are timing out
```

Comment:
Should this and the `condition: false` below  be reverted before merging?

Reply:
Yes to both, they're temporary changes while I figure out why the tests hang in CI but not on my local.